app/v2/models/request/MessageType.scala

```diff
@@ -32,10 +32,44 @@ sealed abstract class DepartureMessageType(val code: String) extends MessageType
 }
 
 object MessageType {
-  case object DepartureDeclaration extends DepartureMessageType("IE015")
+  case object IE004 extends DepartureMessageType("IE004")
+  case object IE009 extends DepartureMessageType("IE009")
+  case object IE013 extends DepartureMessageType("IE013")
+  case object IE014 extends DepartureMessageType("IE014")
+  case object IE015 extends DepartureMessageType("IE015")
+  case object IE019 extends DepartureMessageType("IE019")
+  case object IE028 extends DepartureMessageType("IE028")
+  case object IE029 extends DepartureMessageType("IE029")
+  case object IE035 extends DepartureMessageType("IE035")
+  case object IE045 extends DepartureMessageType("IE045")
+  case object IE051 extends DepartureMessageType("IE051")
+  case object IE054 extends DepartureMessageType("IE054")
+  case object IE055 extends DepartureMessageType("IE055")
+  case object IE056 extends DepartureMessageType("IE056")
+  case object IE060 extends DepartureMessageType("IE060")
+  case object IE170 extends DepartureMessageType("IE170")
+  case object IE906 extends DepartureMessageType("IE906")
+  case object IE928 extends DepartureMessageType("IE928")
```

Comment:
We've not used this pattern for the naming of message types elsewhere - the reason being that this completely hides the purpose of the message, and having that context and understanding has value. If you're going to introduce all of these (and I'm not sure this ticket requires it) then use a meaningful name for each.

Reply:
> and I'm not sure this ticket requires it

Unfortunately, it does, at least in the current state. Any of these message types could be sent via the trader or received from ERMIS, which is why I added them now -- if one of these were missing then we get the 500 error.

The other way I was looking at handling this was just to not convert the message type we receive from transit-movements and leave it as a string. It does mean we need to trust ourselves but... maybe that's what we should do?

Reply:
I've renamed all the constants. The `DepartureMessageType` thing needs to be re-evaluated, but that's not for this PR.